misc/scripts/create-change-note.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Creates a change note and opens it in VSCode for editing.
+
+# Expects to receive the following arguments:
+# - What language the change note is for
+# - Whether it's a query or library change (the string `src` or `lib`)
+# - The name of the change note (in kebab-case)
+# - The category of the change.
+
+# The change note will be created in the `{language}/ql/lib/change-notes` directory.
```

Comment:
Nit: `{language}/ql/lib/change-notes` you might want to replace `lib` in that string.

Reply:
Done. Thanks!